front/src/modules/pathfinding/components/Itinerary/DisplayItinerary/Vias.tsx

```diff
@@ -37,7 +45,7 @@ const Vias = ({ zoomToFeaturePoint, shouldManageStopDuration }: ViasProps) => {
       <Droppable droppableId="droppableVias">
         {(provided) => (
           <div {...provided.droppableProps} ref={provided.innerRef}>
-            {vias.map((via, index) => (
+            {pathSteps.map((via, index) => (
```

Comment:
Why do you use pathSteps instead of vias here ?

Reply:
I struggled yesterday with the Vias component so had to do with the PathSteps but indeed, today (with a clearer mind 😁) there is no problem using vias.